links.css

```diff
@@ -0,0 +1,300 @@
+	:root {
+		--button-background: #1e68d8;
+		--button-border: black;
+		--button-text-color: yellow;
+		--custom-text-color: yellow;
+		--text-color: yellow;
+		--logout-color: yellow;
+		--logout-text-color: #1e68d8;
+		--button-text-shadow-color: #edd400;
+		--button-text-shadow-hover-color: #e0d9ff;
+		--custom-color: red;
+		--fonts: 'Open Sans Bold', Arial, Helvetica , sans-serif;
+		--text-shadow: #aaa;
+		--body-animation-name: polychrome;
+		--body-border-color: black;
+		--animation-color-1: red;
+		--animation-color-2: red;
+		--animation-color-3: gray;
+		--animation-color-4: orange;
+		--animation-color-5: blue;
+		--animation-color-6: red;
+		--gradient-1: rgba(46,0,255,1) 0%;
+		--gradient-2: rgba(218,255,0,1) 15%;
+		--gradient-3: rgba(255,0,0,1) 30%;
+		--gradient-4: rgba(46,0,255,1) 45%;
+		--gradient-5: rgba(218,255,0,1) 60%;
+		--gradient-6: rgba(255,0,0,1) 75%;
+		--gradient-7: rgba(46,0,255,1) 90%;
+		--gradient-8: rgba(218,255,0,1) 100%;
+		--box-shadow-color: black;
+		--box-border-color: black;
+		--checkbox-background: #1e68d8;
+	}
+
+	.button {
+		background-color: var(--button-background);
+		border: solid;
+		border-color: var(--button-border);
+		border-width: thick;
+		border-radius: 10px;
+		color: var(--button-text-color);
+		padding: 15px 32px;
+		text-align: center;
+		display: inline-block;
+		font-size: 19px;
+		margin: 4px 2px;
+		font-weight: bold;
```

Comment:
We will need to get the bold sorted out, because it isn't working with the newly generated files.